code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -69,22 +69,29 @@
 	return
 
 /obj/item/mmi/posibrain/transfer_identity(var/mob/living/carbon/H)
-	name = "positronic brain ([H])"
+	name = "Positronic brain ([H])"
```

Comment:
I feel like this should not be a proper noun, "He is holding Positronic brain (Bob)" vs "He is holding the positronic brain (Bob)"